examples/http/nginx_status.json

```diff
@@ -0,0 +1,36 @@
+{
+    "logo": "nginx.svg",
+    "version": "1.0.0",
+    "name": "Nginx Status",
+    "description": "Monitor Nginx metrics such as: Active, Accepted, and Handled Connections; total requests; etc.",
```

Comment:
the script says 
" You need to read this http://nginx.org/en/docs/http/ngx_http_stub_status_module.html 
  before using this driver "
  not sure if you really **need** to read it, but if it is so important we should include it here